pilot/pkg/networking/plugin/authz/util_test.go

```diff
@@ -170,6 +170,35 @@ func TestConvertToPort(t *testing.T) {
 	}
 }
 
+func TestConvertPortInIntToString(t *testing.T) {
+	testCases := []struct {
+		Name   string
+		V      []int32
+		Expect []string
+		Err    string
+	}{
+		{
+			Name:   "valid ports",
+			V:      []int32{80, 3000, 443},
+			Expect: []string{"80", "3000", "443"},
+		},
+		{
+			Name:   "valid port",
+			V:      []int32{9080},
+			Expect: []string{"9080"},
+		},
```

Comment:
Also add some invalid cases.

Reply:
This is simply converting ints to strings. strconv.Itoa will always succeed if the input type is valid. Are you worried about the range of the ports? If so, we have already checked it before this function is called, right?

Reply:
I see, then we don't need it here

Reply:
Thank you Yangmin